Wbooks/src/app/screens/library/components/bookList/index.js

```diff
@@ -0,0 +1,18 @@
+import React, {Component} from 'react'
+import {
+  View,
+} from 'react-native';
+import BookList from './layout.js';
+
+class BookListContainer extends Component {
+  render() {
+
+    return(
+      <View>
+        <BookList bookList = {this.props.bookList} ></BookList>
+      </View>
```

Comment:
Ok, when you are returning just one component like in this case (just need to return BookList`) there is no need on wrap it on a View. The global rule is that the return statement can return just one component. On the other hand every View without styles means a component that is gonna be rendered without styles.